content/body/log.php

```diff
@@ -7,7 +7,7 @@
 </p>
 
 <p>
-    I have noticed that some screen reader/browser combinations (like NVDA with Firefox) don't read out just the changes: they will read out the contents of the node in its entirety if something is added to the bottom.  If the contents of the region is large, this defeats the 
+    I have noticed that some screen reader/browser combinations (like NVDA with Firefox) don't read out just the changes: they will read out the contents of the node in its entirety if something is added to the bottom.  If the contents of the region large, this defeats the 
```

Comment:
The removal of the "is" word makes the sentence not make sense anymore.
```suggestion
    I have noticed that some screen reader/browser combinations (like NVDA with Firefox) don't read out just the changes: they will read out the contents of the node in its entirety if something is added to the bottom.  If the contents of the region is large, this defeats the 
```